workflow-v2.4.1-e2e/tpl/generate_params.toml

```diff
@@ -0,0 +1,5 @@
+[e2e]
+org = "deis"
+dockerTag = "v2.4.0"
```

Comment:
You mean leave as 2.4.0? I didn't make a new one since originally we talked about not doing e2e but turns out it was a requirement of sorts